lgc/patch/VertexFetch.cpp

```diff
@@ -390,6 +403,24 @@ bool LowerVertexFetch::runImpl(Module &module, PipelineState *pipelineState) {
   if (vertexFetches.empty())
     return false;
 
+  if (pipelineState->getOptions().enableUberFetchShader) {
+    std::unique_ptr<lgc::Builder> desBuilder(Builder::createBuilderImpl(pipelineState->getLgcContext(), pipelineState));
+    desBuilder->setShaderStage(ShaderStageVertex);
+    desBuilder->SetInsertPoint(&(*vertexFetches[0]->getFunction()->front().getFirstInsertionPt()));
+    auto desc = desBuilder->CreateLoadBufferDesc(InternalDescriptorSetId, FetchShaderInternalBufferBinding,
+                                                 desBuilder->getInt32(0), 0, desBuilder->getInt8Ty());
+    desBuilder->CreateInvariantStart(desc);
```

Comment:
Yes, it is a plain 64-bit pointer, we can load the `UberFetchShaderAttribInfo` with `s_load_b128`. 

The IR should be like that you provided if there is no resource mapping, otherwise we will have to use `spillTable`. The code of new method will be similar as `CreateLoadBufferDesc`. In the previous commit, `decset = -1` is for here, what do you think if I let it return early without `buildBufferCompactDesc`.

Reply:
Can you add `BufferFlagAddress` to make `CreateLoadBufferDesc` return an `i64` address of the descriptor? It would trigger an assertion if the flag is used and the descriptor is not compact.